src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj

```diff
@@ -28,5 +29,5 @@
     <InternalsVisibleTo Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication.Tests" />
     <InternalsVisibleTo Include="BasicTestApp" />
   </ItemGroup>
-
+  
```

Comment:
VS auto-formatting coming in to ruin my day once again.

I'll push a fix for this in another commit if I get feedback on this PR so I don't clog the build machines unnecessarily.